src/test/java/org/zipcodewilmington/lab1/StringUtilities.java

```diff
@@ -0,0 +1,21 @@
+package org.zipcodewilmington.lab1;
+
+import java.util.HashSet;
+
+public class StringUtilities {
+
+    public static String unique(String value) {
+
+        HashSet<Character> hash = new HashSet<>();
```

Comment:
You're headed to the right direction. You need something to add the characters back together.